src/Application/src/Calculator.cpp

```diff
@@ -0,0 +1,246 @@
+////////////////////////////////////////////////////////////////////////////////
+/**
+* @file Calculator.cpp
+* @brief Function definitions and class for calculator
+*/
+////////////////////////////////////////////////////////////////////////////////
+
+#include "Calculator.h"
+#include <stdlib.h>
+#include <iostream>
+#include <typeinfo>
+#include "CalculatorMessages.h"
+#include <sstream>
+namespace calculator{
+
+    enum class CalculateStates 
+    {
+	FindingNumber1,
+	OperatorFound,
+	FindingNumber2
+    };
+
+
+//*************/
+// Calculator public methods /
+///
+   
+    std::string fullEquation;
```

Comment:
make this a private member. Right now this is a "global"-ish variable... it would be available to everywhere in the calculator namespace.